app/src/test/java/com/example/weathertimeapp/presenterTest/WeatherDetailsPresenterTest.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.weathertimeapp.presenterTest
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.example.weathertimeapp.data.entity.Day
+import com.example.weathertimeapp.mvp.contract.WeatherDetailsContract
+import com.example.weathertimeapp.mvp.presenter.WeatherDetailsPresenter
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import io.reactivex.Observable
+import io.reactivex.Scheduler
+import io.reactivex.android.plugins.RxAndroidPlugins
+import io.reactivex.disposables.Disposable
+import io.reactivex.internal.schedulers.ExecutorScheduler
+import io.reactivex.plugins.RxJavaPlugins
+import org.junit.Before
+import org.junit.BeforeClass
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+import java.util.concurrent.Executor
+import java.util.concurrent.TimeUnit
+
+class WeatherDetailsPresenterTest {
```

Comment:
the tests are good but try to add some asserts to check de data that the methods handle not only if the some method is called 😉

Reply:
Added asserts to check the data, thanks